playbooks/ansible/roles/samba.setup/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: Create Samba share directories
   file:
-    path: "{{ config.paths.mount }}/{{ item.cluster_volume }}"
+    path: "{{ config.paths.mount }}/{{ item }}"
```

Comment:
Here `{{ item }}` is a dictionary, right?

Reply:
Yes

Reply:
True. I'll fix it.